comixed-model/src/main/java/org/comixedproject/model/comic/Comic.java

```diff
@@ -337,7 +337,9 @@
   @JsonView({View.ComicList.class, View.LibraryUpdate.class})
   private Integer duplicateCount;
 
-  @ManyToMany(mappedBy = "comics", cascade = CascadeType.ALL)
+  @ManyToMany(
+      mappedBy = "comics",
+      cascade = {CascadeType.PERSIST, CascadeType.MERGE})
```

Comment:
Is there a technical benefit to limiting the cascade to only PERSIST and MERGE, implicitly excluding DETACH, REMOVE, etc?

Reply:
I have a bug and I try something but it doesn't work (break an unit test). CascadeType.ALL is better.